### PR TITLE
Fix dependencies for React 15

### PR DESCRIPTION
**Pull request by enricostano**
:boom:

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+Object.assign = null;
+Object.assign = require('object-assign');
```
Comment (andresgutgon) [inline src/config.js:4]: Put an explanation here. So one day we can get rid of it

**Review by andresgutgon — CHANGES_REQUESTED** [review]

**Comment by andresgutgon** [comment]
👍

File: src/config.js (hunk 1 above)
Comment (enricostano) [reply src/config.js:4]: Done

File: src/config.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// React 15 uses native Object.assign, but due to a V8 bug we still need to use the polyfill
+// https://github.com/facebook/react/issues/6451
 Object.assign = null;
```
Comment (andresgutgon) [inline src/config.js:3]: Nice 👏

**Review by andresgutgon — APPROVED** [review]

**Comment by enricostano** [comment]
Merge?

**Pull request merged**